tests/compare-viewer-reports

```diff
@@ -0,0 +1,218 @@
+#! /usr/bin/env python3
+
+# Set emacs mode to Python
+# -*- mode: Python;-*-
+
+"""Compare sets of reports produced by cbmc-viewer."""
+
+from pathlib import Path
+import argparse
+import logging
+import subprocess
+import sys
+
+RESULT_JSON = 'viewer-result.json'
+SOURCE_JSON = 'viewer-source.json'
+OTHER_JSON = [
+    'viewer-coverage.json',
+    'viewer-loop.json',
+    'viewer-property.json',
+    'viewer-reachable.json',
+    'viewer-symbol.json',
+    'viewer-trace.json'
+]
```

Comment:
What if this list grows? Do we warn the user if we find `json` files not listed in here?

Reply:
Added a warning.